### ISSUE TITLE
Build procedure Fedora

### ISSUE THREAD

**Issue by theonlydoo** [darlinghq/darling#234]
Hi,

an ELI5 build procedure for Fedora Latest would be nice!

**Comment by bugaevc** [comment]
It shouldn't be any different than for other distros (it's CMake after all), save for  necessary packages. I can look into this in the evening.

**Comment by theonlydoo** [comment]
It'd be super nice thx

**Comment by theonlydoo** [comment]
so far i had to install :
* bison-devel.x86_64
* bison.x86_64
* flex-devel.x86_64
* xz-libs-5.2.2-2.fc24.x86_64

**Comment by theonlydoo** [comment]
and had a fail @ 45% build :

```[ 45%] Built target libsyscall
Makefile:149 : la recette pour la cible « all » a échouée
```

**Comment by bugaevc** [comment]
> (n.b. : I'm trying to build Darling @ Fedora to try and use 1password Family dmg file, since Windows version is sucky through Wine)

You must know that at this point Darling doesn't yet support running graphical apps.

**Comment by X0rg** [comment]
> and had a fail @ 45% build :
> ```
> Makefile:149` : la recette pour la cible « all » a échouée
> ```
This is a generic error message and it doesn't help us.
We need lines before this error. Try to re-run make with `LC_ALL=C make` (not all of us speak french ;)).

**Comment by ahyattdev** [comment]
Also try `LC_ALL=C make VERBOSE=1`

**Comment by bugaevc** [comment]
Let's just wait until I get home and figure it out ;)

**Comment by X0rg** [comment]
BTW, the TC build successfully passed.

**Comment by bugaevc** [comment]
Added the necessary packages list to README for the latest Fedora 25. `i386` won't work as of now because Debian-specific `LD_LIBRARY_PATH` is currently hardcoded (see #231 & #232), so I'm not bothering to find out what's needed there.

> You must know that at this point Darling doesn't yet support running graphical apps.

When Darling is useful for the end user it'll be just `sudo dnf install darling` ;)
